pkg/daemon/ceph/client/pool.go

```diff
@@ -357,6 +357,12 @@ func CreateECPoolForApp(context *clusterd.Context, clusterInfo *ClusterInfo, poo
 		return errors.Wrapf(err, "failed to create EC pool %s. %s", poolName, string(output))
 	}
 
+	args = []string{"pool", "init", poolName}
```

Comment:
Is `pool init` idempotent?

Reply:
AFAIK and according to my tests, `pool init` is idempotent. 

I'll let @idryomov confirm this (and please review the pr too thanks).